frontend/src/pages/deviation.tsx

```diff
@@ -133,6 +134,68 @@ const OpenInID: FC<{
     </Button>
   );
 };
+
+const Download: FC<{
+  deviation: Pick<DeviationRow, "id" | "suggested_geom" | "suggested_tags"> & {
+    dataset: { provider: { name: string } | null; short_name: string } | null;
+  };
+}> = ({ deviation }) => {
+  const [opened, { open, close }] = useDisclosure();
+  return (
+    <>
+      <Button fullWidth onClick={open}>
+        Hämta...
+      </Button>
+      <Modal opened={opened} onClose={close} title="Ladda ner" centered>
+        <GetGeoJson deviation={deviation} />
+      </Modal>
+    </>
+  );
+};
+
+function geoJsonBlobUrl(geometryOrFeature: string) {
+  const blob = new Blob([geometryOrFeature], { type: "application/geo+json" });
+  return URL.createObjectURL(blob);
+}
+
+const GetGeoJson: FC<{
+  deviation: Pick<DeviationRow, "id" | "suggested_geom" | "suggested_tags"> & {
+    dataset: { provider: { name: string } | null; short_name: string } | null;
+  };
+}> = ({ deviation }) => {
+  const suggested = geojson.readGeometry(deviation.suggested_geom);
+  console.log(suggested);
+  const geom = suggested.transform("EPSG:3006", "EPSG:4326");
+
+  const geojson_url = geoJsonBlobUrl(geojson.writeGeometry(geom));
```

Comment:
snake_case - > camelCase (gäller även raden nedanför) 
```suggestion
  const geojsonUrl = geoJsonBlobUrl(geojson.writeGeometry(geom));
```